### PR TITLE
Remove unnecessary 'ignore...'.

### PR DESCRIPTION
**Pull request by polina-c**

### CODE REVIEW

File: packages/devtools_app/lib/src/charts/flame_chart.dart [hunk 1]
```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// ignore_for_file: import_of_legacy_library_into_null_safe, unnecessary_null_comparison
+// ignore_for_file: unnecessary_null_comparison
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/charts/flame_chart.dart:5]: are we able to get rid of this one too?

File: packages/devtools_app/lib/src/screens/inspector/layout_explorer/box/box.dart [hunk 2]
```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// ignore_for_file: avoid_redundant_argument_values, import_of_legacy_library_into_null_safe, unnecessary_import
+// ignore_for_file: unnecessary_import
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/screens/inspector/layout_explorer/box/box.dart:5]: let's remove this ignore and just get rid of the unnecessary import

File: packages/devtools_app/lib/src/service/service_extension_manager.dart [hunk 3]
```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// ignore_for_file: import_of_legacy_library_into_null_safe, unnecessary_null_comparison
+// ignore_for_file: unnecessary_null_comparison
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/service/service_extension_manager.dart:5]: same question

File: packages/devtools_app/test/instance_viewer/instance_viewer_test.dart [hunk 4]
```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// ignore_for_file: avoid_redundant_argument_values, false positive on required nullable parameters, import_of_legacy_library_into_null_safe
+// ignore_for_file: false positive on required nullable parameters
```
Comment (kenzieschmoll) [inline packages/devtools_app/test/instance_viewer/instance_viewer_test.dart:5]: can we remove?

File: packages/devtools_app/test/matchers/matchers.dart [hunk 5]
```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// ignore_for_file: implementation_imports, import_of_legacy_library_into_null_safe
+// ignore_for_file: implementation_imports
```
Comment (kenzieschmoll) [inline packages/devtools_app/test/matchers/matchers.dart:5]: same question

File: packages/devtools_app/test/span_parser_test.dart [hunk 6]
```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// ignore_for_file: import_of_legacy_library_into_null_safe, unused_import
+// ignore_for_file: unused_import
```
Comment (kenzieschmoll) [inline packages/devtools_app/test/span_parser_test.dart:5]: remove

File: packages/devtools_app/test/test_infra/flutter_test_driver.dart [hunk 7]
```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// ignore_for_file: implementation_imports, import_of_legacy_library_into_null_safe
+// ignore_for_file: implementation_imports
```
Comment (kenzieschmoll) [inline packages/devtools_app/test/test_infra/flutter_test_driver.dart:5]: ditto

File: packages/devtools_app/test/test_infra/flutter_test_environment.dart [hunk 8]
```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// ignore_for_file: implementation_imports, import_of_legacy_library_into_null_safe
+// ignore_for_file: implementation_imports
```
Comment (kenzieschmoll) [inline packages/devtools_app/test/test_infra/flutter_test_environment.dart:5]: ditto

File: packages/devtools_app/test/test_utils/fake_inspector_tree.dart [hunk 9]
```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// ignore_for_file: implementation_imports, import_of_legacy_library_into_null_safe
+// ignore_for_file: implementation_imports
```
Comment (kenzieschmoll) [inline packages/devtools_app/test/test_utils/fake_inspector_tree.dart:5]: ditto

File: packages/devtools_app/test/version_test.dart [hunk 10]
```diff
@@ -3,8 +3,6 @@
 // found in the LICENSE file.
 
 // Ignoring redundant argument values makes the test easier to read.
```
Comment (kenzieschmoll) [inline packages/devtools_app/test/version_test.dart:5]: Since we removed the ignore, remove this comment that goes with it as well.

**Review by kenzieschmoll — COMMENTED** [review]
some comments around whether we can remove remaining ignore_for_file statements

**Review by kenzieschmoll — APPROVED** [review]
Fine to address comments in a follow up

File: packages/devtools_app/lib/src/charts/flame_chart.dart (hunk 1 above)
Comment (polina-c) [reply packages/devtools_app/lib/src/charts/flame_chart.dart:5]: will resolve in other PR

File: packages/devtools_app/lib/src/screens/inspector/layout_explorer/box/box.dart (hunk 2 above)
Comment (polina-c) [reply packages/devtools_app/lib/src/screens/inspector/layout_explorer/box/box.dart:5]: will resolve in other PR

Comment (polina-c) [reply packages/devtools_app/lib/src/screens/inspector/layout_explorer/box/box.dart:5]: Addressed in https://github.com/flutter/devtools/pull/4011

File: packages/devtools_app/lib/src/service/service_extension_manager.dart (hunk 3 above)
Comment (polina-c) [reply packages/devtools_app/lib/src/service/service_extension_manager.dart:5]: Addressed in https://github.com/flutter/devtools/pull/4011

File: packages/devtools_app/test/instance_viewer/instance_viewer_test.dart (hunk 4 above)
Comment (polina-c) [reply packages/devtools_app/test/instance_viewer/instance_viewer_test.dart:5]: done in https://github.com/flutter/devtools/pull/4011

File: packages/devtools_app/test/matchers/matchers.dart (hunk 5 above)
Comment (polina-c) [reply packages/devtools_app/test/matchers/matchers.dart:5]: done in https://github.com/flutter/devtools/pull/4011

File: packages/devtools_app/test/test_infra/flutter_test_driver.dart (hunk 7 above)
Comment (polina-c) [reply packages/devtools_app/test/test_infra/flutter_test_driver.dart:5]: done in https://github.com/flutter/devtools/pull/4011

File: packages/devtools_app/test/test_infra/flutter_test_environment.dart (hunk 8 above)
Comment (polina-c) [reply packages/devtools_app/test/test_infra/flutter_test_environment.dart:5]: done in https://github.com/flutter/devtools/pull/4011

File: packages/devtools_app/test/test_utils/fake_inspector_tree.dart (hunk 9 above)
Comment (polina-c) [reply packages/devtools_app/test/test_utils/fake_inspector_tree.dart:5]: done

File: packages/devtools_app/test/version_test.dart (hunk 10 above)
Comment (polina-c) [reply packages/devtools_app/test/version_test.dart:5]: done in https://github.com/flutter/devtools/pull/4011

**Pull request merged**